src/views/templates/details/tabs/scripts/tests/mocks.ts

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+import { V1Template } from '@kubevirt-ui/kubevirt-api/console';
+
+export const getMockTemplate = (): V1Template =>
+  JSON.parse(JSON.stringify(require('./data/template.json')));
```

Comment:
if i remember correctly, json.* may throw exception, if so, please surround with try catch block

Reply:
Yes. But it's a function for testing. If thrown an exception because there is some issue, is a good thing right?

Reply:
the test should be checking the logic of components if JSON fails to parse because data integrity that is hard coded locally isn't related to the test itself :) 
but also, because its belongs to the test and it's hard coded I don't mind leaving it as it.. :) so up to u...